### PR TITLE
ignore tombstone store.

### PR DESCRIPTION
**Pull request by huachaohuang**

### CODE REVIEW

**Comment by siddontang** [comment]
CI failed @BusyJay

File: src/server/resolve.rs [hunk 1]
```diff
@@ -133,3 +137,90 @@ impl StoreAddrResolver for MockStoreAddrResolver {
         unimplemented!();
     }
 }
+
+#[cfg(test)]
+mod tests {
```
Comment (siddontang) [inline src/server/resolve.rs]: can we mock a PD server in pd mod, so we don't need to start a real PD server in CI test.

Comment (huachaohuang) [inline src/server/resolve.rs]: We can mock `PdClient` for tests, and we already have a `TestPdClient` in `tests/raftstore/pd.rs`.

Comment (siddontang) [inline src/server/resolve.rs]: What I mean is not start PD server in travis CI for testing pd mod.

Comment (siddontang) [inline src/server/resolve.rs]: we can add fire an issue later if we can mock.

**Comment by siddontang** [comment]
LGTM

**Comment by BusyJay** [comment]
LGTM

**Pull request merged**